Add Primeur Blog to staging (#3322)
OpenLiberty/blogs: posts/2023-09-13-Primeur.adoc

author_github: https://github.com/GraceJansen
seo-title: Open Liberty in Action - Primeur’s data integration platform - OpenLiberty.io
seo-description: Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility.
blog_description: "Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility."
open-graph-image: https://openliberty.io/img/twitter_card.jpg
open-graph-image-alt: Open Liberty Logo
---
= Open Liberty in Action - Primeur’s data integration platform
Grace Jansen <https://github.com/GraceJansen>
:imagesdir: /
:url-prefix:
:url-about: /
//Blank line here is necessary before starting the body of the post.

link:https://www.primeur.com/en/[Primeur], an independent software vendor (ISV) based in Italy, focuses on providing smart data integration. They embed Open Liberty Java runtime in their flagship data integration platform, DATA ONE. Open Liberty enables Primeur to make the embedded Java runtime invisible to their data integration customers, while still observable to Primeur’s engineers to be able to support them.

> For over 35 years, Primeur have been helping companies from all over the world to accelerate their data integration projects, ensuring their customers reach their business goals while making the most of their existing IT investments and avoiding unnecessary expenses. To help their clients achieve this, Primeur established their flagship data integration platform, link:https://www.primeur.com/data-one[DATA ONE], in 2020 which is designed to federate different technologies, platforms, data types and deployment models. Primeur embed the Open Liberty Java runtime in DATA ONE, using it to power the core data integration choreographies. 

Reviewer: ```suggestion
For over 35 years, Primeur have been helping companies from all over the world to accelerate their data integration projects, ensuring their customers reach their business goals while making the most of their existing IT investments and avoiding unnecessary expenses. To help their clients achieve this, Primeur established their data integration platform, link:https://www.primeur.com/data-one[DATA ONE], in 2020 which is designed to federate different technologies, platforms, data types and deployment models. Primeur embed the Open Liberty Java runtime in DATA ONE, using it to power the core data integration choreographies. 
```

== Why choose Open Liberty?

Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem like a contradiction and might not be something you have considered when selecting a cloud-native runtime, these two factors are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty (whether on-prem or in a cloud environment) to run its business application.

== Invisibility

The great benefit of Open Liberty’s ability to remain invisible to their clients, is that Primeur can embed, completely configure, and operate Open Liberty within their DATA ONE product without their customers having to be aware that Open Liberty is even there. Their customers need only care about DATA ONE. So customers’ developers don’t have to skill up on yet another technology or care about how to operate the runtime that their platform is based upon.

== Observability

> In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, Primeur are aware that shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning of what can be a long journey that can last several years until the next major product version upgrade. During this time, customers raise support requests to investigate problems that arise in the product but whose root cause can often be challenging to identify.

Reviewer: ```suggestion
In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning. When customer's raise problems, the complexity of cloud-native application architectures and distributed cloud infrastructure can make it difficult to detect bottlenecks and failure points. Primeur's engineers are able to effectively and efficiently use Open Liberty's problem-determination tools to monitor, trace, and generally probe Open Liberty until they are able to identify the root cause of any problems reported by their customers. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.

```

> In the ever-increasing complexity of application architecture, cloud-native tooling and distributed cloud infrastructure, bottlenecks and failure points can be very remote and well-hidden making them especially difficult to detect. It is vitally important for Primeur to be able to diagnose any problems the occur as quickly as possible to limit the impact on their customer’s business. This is why observability is critical for Primeur. Their support team needs to be able to effectively and efficiently monitor, trace and generally probe Open Liberty until they are able to identify the root cause of the problem that the customer is facing. Open Liberty offers a wide range of problem-determination tools and techniques to enable Primeur to achieve this. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.

Reviewer: ```suggestion
In the ever-increasing complexity of application architecture, cloud-native tooling and distributed cloud infrastructure, bottlenecks and failure points can be very remote and well-hidden making them especially difficult to detect. It is vitally important for Primeur to be able to diagnose any problems that occur as quickly as possible to limit the impact on their customer’s business. This is why observability is critical for Primeur. Their support team needs to be able to effectively and efficiently monitor, trace and generally probe Open Liberty until they are able to identify the root cause of the problem that the customer is facing. Open Liberty offers a wide range of problem-determination tools and techniques to enable Primeur to achieve this. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.
```

> In the ever-increasing complexity of application architecture, cloud-native tooling and distributed cloud infrastructure, bottlenecks and failure points can be very remote and well-hidden making them especially difficult to detect. It is vitally important for Primeur to be able to diagnose any problems the occur as quickly as possible to limit the impact on their customer’s business. This is why observability is critical for Primeur. Their support team needs to be able to effectively and efficiently monitor, trace and generally probe Open Liberty until they are able to identify the root cause of the problem that the customer is facing. Open Liberty offers a wide range of problem-determination tools and techniques to enable Primeur to achieve this. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.

Reviewer: ```suggestion

```

Reviewer: (ie remove this para and just have a single condensed para in the Observability section)

---
layout: post
title: "Open Liberty in Action - Primeur’s data integration platform"
# Do NOT change the categories section
categories: blog
author_picture: https://avatars3.githubusercontent.com/GraceJansen
author_github: https://github.com/GraceJansen
seo-title: Open Liberty in Action - Primeur’s data integration platform - OpenLiberty.io
seo-description: Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility.
blog_description: "Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility."
open-graph-image: https://openliberty.io/img/twitter_card.jpg
open-graph-image-alt: Open Liberty Logo
---
= Open Liberty in Action - Primeur’s data integration platform
Grace Jansen <https://github.com/GraceJansen>
:imagesdir: /
:url-prefix:
:url-about: /
//Blank line here is necessary before starting the body of the post.

> link:https://www.primeur.com/en/[Primeur], an independent software vendor (ISV) based in Italy, focuses on providing smart data integration. They embed Open Liberty Java runtime in their flagship data integration platform, DATA ONE. Open Liberty enables Primeur to make the embedded Java runtime invisible to their data integration customers, while still observable to Primeur’s engineers to be able to support them.

Reviewer 2: ```suggestion
link:https://www.primeur.com/en/[Primeur], an independent software vendor (ISV) based in Italy, focuses on providing smart data integration. They embed  the Open Liberty Java runtime in their flagship data integration platform, DATA ONE. Open Liberty enables Primeur to make the embedded Java runtime invisible to their data integration customers, while still observable to the Primeur engineers who support them.
```

> For over 35 years, Primeur have been helping companies from all over the world to accelerate their data integration projects, ensuring their customers reach their business goals while making the most of their existing IT investments and avoiding unnecessary expenses. To help their clients achieve this, Primeur established their data integration platform, link:https://www.primeur.com/data-one[DATA ONE], in 2020 which is designed to federate different technologies, platforms, data types and deployment models. Primeur embed the Open Liberty Java runtime in DATA ONE, using it to power the core data integration choreographies. 

Reviewer 2: ```suggestion
For over 35 years, Primeur have helped companies from all over the world to accelerate their data integration projects, ensuring their customers reach their business goals while making the most of their existing IT investments and avoiding unnecessary expenses. To help their clients achieve this, in 2020 Primeur established their data integration platform, link:https://www.primeur.com/data-one[DATA ONE], which is designed to federate different technologies, platforms, data types and deployment models. Primeur embed the Open Liberty Java runtime in DATA ONE, using it to power the core data integration choreographies. 
```

== Why choose Open Liberty?

> Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem like a contradiction and might not be something you have considered when selecting a cloud-native runtime, these two factors are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty (whether on-prem or in a cloud environment) to run its business application.

Reviewer 2: ```suggestion
Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem contradictory and might not be something you have considered when selecting a cloud-native runtime, they are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty to run its business application, whether on-prem or in a cloud environment.
```

image::img/blog/primeur_logo.png[Primeur Logo ,width=70%,align="center"]

== Invisibility

> The great benefit of Open Liberty’s ability to remain invisible to their clients, is that Primeur can embed, completely configure, and operate Open Liberty within their DATA ONE product without their customers having to be aware that Open Liberty is even there. Their customers need only care about DATA ONE. So customers’ developers don’t have to skill up on yet another technology or care about how to operate the runtime that their platform is based upon.

Reviewer 2: ```suggestion
The great benefit of Open Liberty’s ability to remain invisible so that Primeur can embed, completely configure, and operate Open Liberty within their DATA ONE product without their customers knowing that Open Liberty is even there. Their customers need only care about DATA ONE. So customers’ developers don’t have to skill up on yet another technology or care about how to operate the runtime that their platform is based upon.
```

== Observability

> In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning. When customers raise problems, the complexity of cloud-native application architectures and distributed cloud infrastructure can make it difficult to detect bottlenecks and failure points. Primeur's engineers are able to effectively and efficiently use Open Liberty's problem-determination tools to monitor, trace, and generally probe Open Liberty until they are able to identify the root cause of any problems reported by their customers. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also diagnose problems at the JVM level in a much more efficient and easier way too.

Reviewer 2: ```suggestion
In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning. When customers raise problems, the complexity of cloud-native application architectures and distributed cloud infrastructure can make it difficult to detect bottlenecks and failure points. Primeur's engineers can effectively and efficiently use Open Liberty's problem-determination tools to monitor, trace, and generally probe Open Liberty until they can identify the root cause of any problems reported by their customers. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also easily diagnose problems at the JVM level.
```
